src/Ledger/Epoch.agda

```diff
@@ -72,6 +73,9 @@ record EpochStructure : Set₁ where
     addEpoch : HasAdd Epoch
     addEpoch = record { _+_ = _+ᵉ'_ }
 
+    Dec-≤ᵉ : ∀ {n m} → Dec (n ≤ᵉ m)
+    Dec-≤ᵉ = Decidable²⇒Dec _≤ᵉ?_
```

Comment:
Something like this should have been part of my type classes PR, but okay...

Also, what's the point of `Decidable²⇒Dec`?  Why don't we use `Decidable` directly?
```agda
    Dec-≤ᵉ : ∀ {n m} → Dec (n ≤ᵉ m)
    Dec-≤ᵉ {n}{m} = n ≤ᵉ? m
```

Reply:
Yes, but your PR is not even out of WIP yet, and I needed this now so that it can be included in the 0.9 release.

I guess `Dec-≤ᵉ` could have been implemented directly, but this highlights a general pattern that we could use in the future.

Reply:
I'll "unblock" it now.

Reply:
Sorry, I didn't realize you were waiting for me to approve the change.